reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -71,6 +71,7 @@ class FreemarkerTemplateProcessor(
         val dataModel = mapOf(
             "projects" to projects,
             "packages" to packages,
+            "advisorResults" to input.ortResult.getAdvisorResultContainers(),
```

Comment:
Is that so useful given that the user could already call `ortResult.getAdvisorResultContainers()` in the template?

Reply:
I agree we should drop this.